packages/nextjs/src/app-router/client/ClerkProvider.tsx

```diff
@@ -99,3 +99,36 @@ export const ClientClerkProvider = (props: NextClerkProviderProps) => {
     </ClerkNextOptionsProvider>
   );
 };
+
+const AccountlessCreateKeys = (props: NextClerkProviderProps) => {
+  const { children } = props;
+  const [state, fetchKeys] = React.useActionState(createAccountlessKeysAction, null);
+  useEffect(() => {
+    React.startTransition(() => {
+      fetchKeys();
+    });
+  }, []);
+
+  if (!React.isValidElement(children)) {
+    return children;
+  }
+
+  return React.cloneElement(children, {
+    key: state?.publishableKey,
+    publishableKey: state?.publishableKey,
+    claimAccountlessKeysUrl: state?.claimUrl,
+    __internal_bypassMissingPk: true,
```

Comment:
`__internal_bypassMissingPk` will make `@clerk/react` to not throw when pk is missing, and we use `key` in order to remount the the Provider with the correct keys on those are fetched.

This is only necessary,  because we can't simply render `null` as React requires `<html><body>` to exist on hydration and due to our pattern of `<ClerkProvider> <html> </ClerkProvider>` we cannot guarantee that this will work smoothly